app/main.py

```diff
@@ -1,6 +1,16 @@
 from typing import Callable
 
+cache_dict = {}
+
 
 def cache(func: Callable) -> Callable:
-    # Write your code here
-    pass
+    def inner(*args, **kwargs) -> int:
```

Comment:
You could use a decorator from `functools` to preserve the wrapped function name and docstring